main.tf

```diff
@@ -24,12 +40,11 @@ module "label" {
   # ...name-workers-blue instead of ...name-blue-workers), historically we forced "workers"
   # to the end of the attribute list, so we do it again here to maintain compatibility.
   attributes = compact(concat(module.this.attributes, ["workers"]))
-  tags       = local.tags
+  tags       = var.tags
```

Comment:
`tags = local.tags` was needed to add the extra tags. `tags = var.tags` is not needed, it is handled by `context`.
```suggestion
```